src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -128,28 +130,32 @@ const ExpandedNodeField: FC<ExpandedNodeFieldProps> = ({ title, description }) =
 
 type StatusFieldProps = {
   documentation: DocumentationResource | undefined;
+  isAccepted: boolean | null;
 };
 
-const StatusField: FC<StatusFieldProps> = ({ documentation }) => {
+const StatusField: FC<StatusFieldProps> = ({ documentation, isAccepted }) => {
   if (!documentation?.resource) {
     return null;
   }
   const status = documentation.status;
   const rawDate = documentation.resource?.meta?.lastUpdated;
-  if (rawDate)
+  if (rawDate) {
+    const date = new Date(rawDate).toLocaleString('en-us');
     return (
       <ExpandedNodeField
         key="Status"
-        title={status}
-        description={new Date(rawDate).toLocaleString('en-us')}
+        title={isAccepted === false ? 'Declined' : status}
```

Comment:
This can be simplified. We can also make sure to capitalize the status with the following:

```suggestion
        title={isAccepted ? status.charAt(0).toUpperCase() + status.slice(1) : 'Declined'}
```